Microsoft.Azure.CosmosRepository/src/Providers/DefaultCosmosItemConfigurationProvider.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) IEvangelist. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Concurrent;
+using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.CosmosRepository.Options;
+
+namespace Microsoft.Azure.CosmosRepository.Providers
+{
+    class DefaultCosmosItemConfigurationProvider : ICosmosItemConfigurationProvider
+    {
+        private static readonly ConcurrentDictionary<Type, ItemOptions> ItemOptionsMap = new();
```

Comment:
```suggestion
        private static readonly ConcurrentDictionary<Type, ItemOptions> s_itemOptionsMap = new();
```

Inconsistent styling, static fields should be `s_` prefixed.